test/integration/progress_int_test.go

```diff
@@ -19,16 +19,17 @@ func (suite *ProgressIntegrationTestSuite) TestProgress() {
 	defer ts.Close()
 
 	cp := ts.Spawn("checkout", "ActiveState-CLI/Empty")
+	cp.Expect("Resolving Dependencies")
+	cp.ExpectRe(`[^.]+?✔ Done`, e2e.RuntimeSolvingTimeoutOpt)
```

Comment:
This first checkout seems to specifically be about asserting that it does NOT use dot progress, but these changes seem to make it so that it is asserting that it DOES use dot progress.

Reply:
This test is expecting that there are no dots between "Resolving Dependencies" and  "✔ Done", so it's asserting that it does NOT use dot progress.

I had to do a few iterations of this test, and during these iterations I found that `cp.Output()` gave me all the escape sequences of the \\, |, and / interactive progress between "Resolving Dependencies" and "✔ Done". I am confident this test is working as intended.

Reply:
Ahh.. the way the regex is written I'd assume it was checking for `... ✔ Done`. I had interpreted the dot as a literal dot rather than a regex rule. Might be worth adding a comment for clarity.